docs/src/content/docs/de/guides/sidebar.mdx

```diff
@@ -0,0 +1,472 @@
+
+```md
+---
+title: Meine Seite
+sidebar:
+  # Setzt eine eigene Beschriftung für den Link
+  label: Benutzerdefinierte Seitenleistenbeschriftung
+  # Legen du eine benutzerdefinierte Reihenfolge für den Link fest (niedrigere Zahlen werden weiter oben angezeigt)
+  order: 2
+  # Fügen du dem Link ein Abzeichen hinzu
+  badge:
+    text: Neu
+    variant: tip
+---
+```
+
+Eine autogenerierte Gruppe, die eine Seite mit dem obigen Frontmatter enthält, erzeugt die folgende Seitenleiste:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Anleitungen',
+			items: [
+				{ label: 'Eine Seite', link: '#' },
+				{
+					label: 'Benutzerdefinierte Seitenleistenbeschriftung',
+					link: '#',
+					badge: { text: 'Neu', variant: 'tip' },
+				},
+				{ label: 'Andere Seite', link: '#' },
+			],
+		},
+	]}
+/>
+
+:::note[Anmerkung]
+Die `sidebar`-Frontmatter-Konfiguration wird nur für automatisch generierte Links verwendet und wird für manuell definierte Links ignoriert.
+:::
+
+## Abzeichen
+
+Links können auch eine `badge`-Eigenschaft enthalten, um ein Abzeichen neben dem Link-Label anzuzeigen.
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Anleitungen',
+			items: [
+				// Ein Link mit einem "Neu"-Abzeichen.
+				{
+					label: 'Komponenten',
+					link: '/guides/components/',
+					badge: 'Neu',
+				},
+			],
+		},
+	],
+});
+```
+
+Die obige Konfiguration erzeugt die folgende Seitenleiste:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Anleitungen',
+			items: [
+				{
+					label: 'Komponenten',
+					link: '/guides/components/',
+					badge: { text: 'Neu', variant: 'default' },
+				},
+			],
+		},
+	]}
+/>
+
+### Abzeichenvarianten
+
+Passe das Design des Abzeichens an, indem du ein Objekt mit den Eigenschaften `text` und `variant` verwendest.
+
+Der `text` steht für den anzuzeigenden Inhalt (z.B. "Neu").
+Überschreibe das `default`-Styling, das die Akzentfarbe deiner Website verwendet, indem du die Eigenschaft `variant` auf einen der folgenden Werte setzt: `note`, `tip`, `danger`, `caution` oder `success`.
+
+```js
+starlight({
+	sidebar: [
+		{
+			label: 'Anleitungen',
+			items: [
+				// Ein Link mit einem gelben "Experimentell"-Abzeichen.
+				{
+					label: 'Komponenten',
+					link: '/guides/components/',
+					badge: { text: 'Experimentell', variant: 'caution' },
+				},
+			],
+		},
+	],
+});
+```
+
+Die obige Konfiguration erzeugt die folgende Seitenleiste:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'Anleitungen',
+			items: [
+				{
+					label: 'Komponenten',
+					link: '/guides/components/',
+					badge: { text: 'Experimentell', variant: 'caution' },
+				},
+			],
+		},
+	]}
+/>
+
+## Internationalisierung
+
```

Comment:
Hier wurde auch einige Details in fd4aa635440b5c6ec945c49423f81f18331ac559 geändert. Kannst du diese einbeziehen?